src/Infrastructure.EntityFramework/Auth/Configurations/GrantEntityTypeConfiguration.cs

```diff
@@ -12,6 +12,10 @@ public void Configure(EntityTypeBuilder<Grant> builder)
             .HasKey(s => s.Id)
             .IsClustered();
 
+        builder
+            .Property(s => s.Id)
+            .UseIdentityColumn();
+
```

Comment:
ℹ️ This is for posterity and has no effect but I wanted to point it out explicitly.